sys/net/pfvar.h

```diff
@@ -1598,7 +1598,7 @@ struct pf_pdesc {
 #ifdef INET6
 		struct icmp6_hdr	icmp6;
 #endif /* INET6 */
-		char any[0];
```

Comment:
__subobject_use_container_bounds?

Reply:
Though really this is extremely pointless a member. One should just be casting a pointer to the union to a char * instead.

Reply:
Given there are only the following four uses in sys that I can find:

```
origin/dev:sys/netpfil/pf/pf.c:      m_copyback(m, off, hdrlen, pd->hdr.any);
origin/dev:sys/netpfil/pf/pf.c:          m_copyback(m, off, hdrlen, pd->hdr.any);
origin/dev:sys/netpfil/pf/pf.c:      m_copyback(m, off, hdrlen, pd->hdr.any);
origin/dev:sys/netpfil/pf/pf.c:          m_copyback(m, off, hdrlen, pd->hdr.any);
```

I'd rather see the code be less dodgy and cast instead. This is something that can be upstreamed.